src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,42 @@
+package duke.task;
+
+import duke.TaskCreationException;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents an event task
+ */
+public class Event extends Task {
+    private final LocalDate from;
```

Comment:
Might want to consider renaming `from` and `to` to `startDate` and `endDate` which could be a little clearer.